applications/RomApplication/python_scripts/rom_nn_trainer.py

```diff
@@ -0,0 +1,260 @@
+import numpy as np
+import os
+import json
+
+import tensorflow as tf
+from keras.models import Model
+from keras import layers
+from keras.optimizers import AdamW
+from keras.callbacks import LearningRateScheduler
+
+import KratosMultiphysics
+
+
+class Rom_NN_trainer(object):
```

Comment:
```suggestion
class RomNeuralNetworkTrainer(object):
```
Please follow the Kratos convention for the module/class names (filename must be modified accordingly to `rom_neural_network_trainer.py`